content/en/docs/instrumentation/php/propagation.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Propagation
+description: Context propagation for the PHP API
+aliases: [/docs/instrumentation/php/api/propagation]
```

Comment:
Drop the alias.

Reply:
You dropped too many aliases. This one (in the new file) needed to be removed, not the one in `content/en/docs/instrumentation/php/getting-started.md`. Restore that alias.